src/components/ModalContent/__stories__/ModalContent.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate, createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+
+export const metaSettings = createStoryMetaSettings({
+  component: ModalContent,
+  enumPropNamesArray: [], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [], // List the component's actions here
+});
+
+<Meta
+  title="Feedback/Modal/ModalContent"
+  component={ ModalContent }
+  argTypes={ metaSettings.argTypes }
+  decorators={ metaSettings.decorators }
+/>
+
+<!--- Component template -->
+
+export const modalContentTemplate = createComponentTemplate(ModalContent);
+
+<!--- Component documentation -->
+
+# ModalContent
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview"
+         args={ {prop1: "prop's default value", prop2: "prop's default value"} }>
+    { modalContentTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ ModalContent } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Usage guideline 1",
+  "Usage guideline 2",
+  "Usage guideline 3",
+]}/>
+
+<Tip title="Not what you were looking for?">Tip content</Tip>
+
+## Variants
+### Story title 1
+Description of story 1
+<Canvas>
+  <Story name="Story title 1">
+    <ModalContent />
+    <ModalContent />
+  </Story>
+</Canvas>
+
+### Story title 2
+Description of story 2
+<Canvas>
+  <Story name="Story title 2">
+    <ModalContent />
+    <ModalContent />
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:<ModalContent />,
+        description:"Best practice description."
+      },
+      negative: {
+        component:<ModalContent />,
+        description:"Bad practice description."
+      }
+    },
+    {
+      positive: {
+        component:<ModalContent />,
+        description:"Best practice description."
+      },
+      negative: {
+        component:<ModalContent />,
+        description:"Bad practice description."
+      }
+    },
+    {
+      positive: {
+        component:<ModalContent />,
+        description:"Best practice description."
+      },
+      negative: {
+        component:<ModalContent />,
+        description:"Bad practice description."
+      }
+    },
+    {
+      positive: {
+        component:<ModalContent />,
+        description:"Best practice description."
+      },
+      negative: {
+        component:<ModalContent />,
+        description:"Bad practice description."
+      }
+    }
+  ]}
+/>
+
+
+## Use cases and examples
+
+### Story title 3
+Description of story 3
+<Canvas>
```

Comment:
Is this story needed? or we can delete it?

Reply:
👍